### DISCUSSION TITLE
How do I make a custom pictorial element?

### DISCUSSION THREAD

**Discussion by msk-nightly** [cdelker/schemdraw#39] [Q&A] [answered]
Hello there, hope this finds you all well.

I've drawn an electric circuit using SchemDraw and now, I want to add an unconventional picture as an element. How can I do this? The circuit is just a model of a mechanical system btw. An example is as follows:
![abcd](https://github.com/cdelker/schemdraw/assets/73292171/d58c5861-abb0-439a-8805-b5577f8ac974)

I simply want to add a pictorial element as seen in the attached image.
Thank you for your time and patience.

**Comment by cdelker** [comment, marked as answer]
You can add images as an element using [`ElementImage`](https://schemdraw.readthedocs.io/en/stable/classes/drawing.html#schemdraw.elements.ElementImage). For simple image elements, the image can just be added to a schemdraw drawing directly:

`elm.ElementImage('myimage.png', width=2, height=2)`

Note width and height are relative to schemdraw's drawing unit, the length of a standard resistor.

For more complicated applications involving anchors, see the documentation and an example [here](https://schemdraw.readthedocs.io/en/stable/elements/images.html#images).

**Reply by msk-nightly** [reply]
That's awesome! Thank you for your time and effort.

**Reply by msk-nightly** [reply]
I've been able to finish my task. Thanks again. 

By the way, another thing: can you kindly enable the option to Download the SchemDraw Documentation as a pdf/epub, if it isn't already?
ReadTheDocs seems to have such a feature but I cannot find an option to download the SchemDraw Docs in those formats.
Thanks in advance.

**Reply by cdelker** [reply]
Sure - I had to update the build config and switch to a different image converter (Latex doesn't like a lot of SVGs), but there are now PDF and EPUB in the [Downloads](https://readthedocs.org/projects/schemdraw/downloads/) section.

**Reply by msk-nightly** [reply]
Awesome! It's a lifesaver to have all of the documentation in one place which is easily accessible/searchable offline. 

Thank you very much for your time, effort and patience.
